### ISSUE TITLE
base href issue

### ISSUE THREAD

**Issue by kukac7** [arkon/ng-inline-svg#50]
get error:
`zone.js:2933 GET http://assets/img/logo.svg net::ERR_NAME_NOT_RESOLVED`

`<span [inlineSVG]="'/assets/img/logo.svg'"></span>`

`<base href="/">`

what is the problem?
previous version was good.
I'm using a hmr.

**Comment by arkon** [comment]
Seems like a regression from #49. I'll need to take a look tonight.

**Comment by dottodot** [comment]
Yes setting `baseUrl` in `InlineSVGModule.forRoot()` not longer works

**Comment by chancezeus** [comment]
Another regression is that this change also makes it impossible to fetch and embed svg icons from two different sources (local and cdn for example). If a baseUrl will always be set (through detection), I would like to suggest that the plugin also checks if the provided source contains a http(s) scheme (ie. `/^https?:\/\//i`), that way local resources can be loaded using the baseUrl set by the developer/detection and cdn sources will still be loaded correctly.

**Comment by arkon** [comment]
Should be fixed in v5.1.1. Please reopen if you're still seeing issues!

**Comment by dottodot** [comment]
Still doesn't work when setting `baseUrl` in `InlineSVGModule.forRoot()` and `svgService.setBaseUrl` no longer seems to be an option

**Comment by kukac7** [comment]
it works well with me!
but I do not use the `baseUrl ` in `InlineSVGModule.forRoot()`.

**Comment by arkon** [comment]
@dottodot What're you expecting/seeing instead?

**Comment by dottodot** [comment]
Before the change (v5.0.0) I set the baseUrl to my cdn and this would be added before anything I added to [inlineSVG]. Now this doesn't happen, so it's trying to fetch the svg relative to my site instead.

**Comment by arkon** [comment]
Can you give an example of the base URL and what you're passing into the directive?

**Comment by dottodot** [comment]
yes it's `https://res.cloudinary.com/cloud-name/image/upload/`

**Comment by dottodot** [comment]
Sorry just realised I only gave you half an answer there.

so the baseUrl is something like `https://res.cloudinary.com/cloud-name/image/upload/`

and I'm using the directive as per below where the `template_url` is file path

```
<svg #template id="template" xmlns="http://www.w3.org/2000/svg" [attr.width]="width" [attr.height]="height" [attr.viewBox]="viewBox"
      [prepend]="false" [inlineSVG]="template_url" [replaceContents]="false" (onSVGInserted)="svgData($event)">
       <defs>
        <linearGradient x1="50%" y1="0%" x2="50%" y2="100%" *ngFor="let colour of customService.getGradients()" [attr.id]="colour.swatch + '-gradient'">
          <stop *ngFor="let gradient of colour.gradients" [attr.stop-color]="gradient.colour" [attr.offset]="gradient.position"></stop>
        </linearGradient>
      </defs>
</svg>
```

**Comment by arkon** [comment]
@dottodot So you're using ng-inline-svg v5.1.1 and `InlineSVGModule.forRoot({ baseUrl: 'https://res.cloudinary.com/cloud-name/image/upload/' })`? It seems to work fine when I'm trying it.

**Comment by dottodot** [comment]
@arkon Think it was because it was in a lazy loaded module. Before I just add `InlineSVGModule.forRoot()` in my app.module and just imported `InlineSVGModule` into my lazy loaded module. Now it seems I have to do `InlineSVGModule.forRoot()` in my lazy loaded module.

**Comment by arkon** [comment]
@dottodot Fixed in v5.1.2.
